docs/developers/developers-setup.md

```diff
@@ -148,7 +148,7 @@ sudo ninja uninstall -C build
 sudo rm -v /usr/local/share/locale/*/LC_MESSAGES/diffuse.mo
 ```
 
-### Setup on Mac OS
+### Setup on macOS using Meson
```

Comment:
Why add "using Meson" here? There are no other build systems for macOS. I'd rather not add this information as it could lead to some confusion.

(I'm ok with "Mac OS" becoming "macOS", I wanted to change that, but since the first PR about Mac wrote Mac OS, I wanted to use the same terms)